BlocSpot/app/src/main/java/com/izor066/android/blocspot/api/model/PointsOfInterest.java

```diff
@@ -0,0 +1,41 @@
+package com.izor066.android.blocspot.api.model;
+
+/**
+ * Created by igor on 22/9/15.
+ */
+public class PointsOfInterest {
+
+    private String title;
```

Comment:
All these fields should be `final` in order to promote immutability. A good read on why immutability is good can be found [here](http://www.yegor256.com/2014/06/09/objects-should-be-immutable.html).